docs/mindsdb_sql/knowledge-bases.mdx

```diff
@@ -215,241 +452,294 @@ SELECT id, content, metadata, embeddings
+Here is the output:
 
 ```sql
-SELECT b.id, b.title, kb.chunk_content, kb.relevance
-FROM books AS b
-LEFT JOIN my_kb AS kb 
-ON b.id = kb.id
-WHERE b.topic = 'fantasy'
-AND kb.content = 'a novel about benevolent ai';
++----------+------------------+------------------------+------------------------+--------------------+
+| order_id | product          | notes                  | chunk_content          | relevance          |
++----------+------------------+------------------------+------------------------+--------------------+
+| A1B      | Wireless Mouse   | Request color: black   | Request color: black   | 0.5106591666649376 |
++----------+------------------+------------------------+------------------------+--------------------+
 ```
-</Accordion>
 
-## Delete from a Knowledge Base
+## `DELETE FROM` Syntax
 
 Here is the syntax for deleting from a knowledge base:
 
 ```sql
-DELETE FROM <kb_name> 
-WHERE <condition>;
+DELETE FROM my_kb
+WHERE id = 'A1B';
 ```
 
 Upon execution, it identifies matching records based on the user-defined condition and removes all associated data (metadata, content, chunks, embeddings) for matching records from the KB's storage.
 
-The following is an explanation of the syntax and parameters:
+## `DROP KNOWLEDGE_BASE` Syntax
 
-* `<kb_name>` is a unique identifier of a knowledge base.
-* `<condition>` is a standard SQL WHERE clause condition operating on metadata columns.
+Here is the syntax for deleting a knowledge base:
 
-<Accordion title="Examples">
+```sql
+DROP KNOWLEDGE_BASE my_kb;
+```
 
-Here are the usage examples:
+Upon execution, it removes the knowledge base with its content.
 
-* Deleting from a knowledge base:
+## Complete Example
+
+Here is the data that will be inserted into the knowledge base.
 
 ```sql
-DELETE FROM my_kb 
-WHERE id IN ('scifi_001', 'scifi_0010', 'bus_89');
++----------+-------------------+------------------------+
+| order_id | product           | notes                  |
++----------+-------------------+------------------------+
+| A1B      | Wireless Mouse    | Request color: black   |
+| 3XZ      | Bluetooth Speaker | Gift wrap requested    |
+| Q7P      | Laptop Stand      | Prefer aluminum finish |
++----------+-------------------+------------------------+
 ```
-</Accordion>
 
-## Drop a Knowledge Base
+Here is how to create a knowledge base specifically for the data.
 
-Here is the syntax for deleting a knowledge base:
+```sql
+CREATE KNOWLEDGE_BASE my_kb
+USING
+    embedding_model = {
+       "provider": "openai",
+       "model_name" : "text-embedding-3-large",
+       "api_key": "sk-abc123"
+    },
+    reranking_model = {
+       "provider": "openai",
+       "model_name": "gpt-4o",
+       "api_key": "sk-abc123"
+    },
+    metadata_columns = ['product'],
+    content_columns = ['notes'],
+    id_column = 'order_id';
+```
+
+Here is how to insert the data.
 
 ```sql
-DROP KNOWLEDGE_BASE <kb_name>;
+INSERT INTO my_kb
+    SELECT order_id, product, notes
+    FROM orders;
```

Comment:
Do we need integration name in the FROM integration.orders?

Reply:
Yes. I'll add this data into our sample db and show users explicitly how to access it.